scripts/exports.sh

```
@@ -0,0 +1,43 @@
+# used to inject env variables into the docker-compose.yml file
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)